### PR TITLE
Silence wal logging during testing

### PR DESCRIPTION
**Pull request by corylanou**

### CODE REVIEW

File: cmd/influxd/run/server_helpers_test.go [hunk 1]
```diff
@@ -183,6 +183,7 @@ func NewConfig() *run.Config {
 
 	c.Data.Dir = MustTempFile()
 	c.Data.WALDir = MustTempFile()
+	c.Data.WALEnableLogging = false
```
Comment (benbjohnson) [inline cmd/influxd/run/server_helpers_test.go]: Can you change it to `WALLoggingEnabled` to be consistent with the other `Enabled` fields?

**Comment by otoolep** [comment]
+1, once suggested changed by @benbjohnson is in place.

**Comment by corylanou** [comment]
@otoolep @benbjohnson this had a bit of a ripple affect...  another quick look pls?

**Comment by benbjohnson** [comment]
+1

**Pull request merged**